platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -64,21 +63,36 @@
 
     private final MBeanServer mBeanServer;
 
+    private final List<ConfigurationMigratable> configurationMigratables;
+
+    private final List<DataMigratable> dataMigratables;
+
     /**
      * Constructor.
      *
      * @param configurationAdminMigration object used to export {@link org.osgi.service.cm.Configuration}
      *                                    objects from {@link org.osgi.service.cm.ConfigurationAdmin}
      * @param mBeanServer                 object used to register this object as an MBean
+     * @param configurationMigratables    list of {@link ConfigurationMigratable} services. Needs
+     *                                    to be kept up-to-date by the client of this class.
+     * @param dataMigratables             list of {@link DataMigratable} services. Needs
+     *                                    to be kept up-to-date by the client of this class.
```

Comment:
My understanding, and @lessarderic feel free to correct me, is that by passing in a reference to this special list through blueprint, the variable always has up-to-date information. We may need to get confirmation on this.